src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -2048,9 +2048,9 @@ export class PostgresStorageAdapter implements StorageAdapter {
     if (where.pattern.length > 0 || !estimate) {
       qs = `SELECT count(*) FROM $1:name ${wherePattern}`;
     } else {
+      await this.updateEstimatedCount(className);
```

Comment:
If I remember correctly, this else statement usually gets hit by the ParseDashboard and maybe the use of the masterKey. Since the dashboard heavily relies on count, do you think this be a noticeable performance hit whenever a database manager (or multiple managers) are logged into the Dashboard?

Reply:
> Since the dashboard heavily relies on count

The dashboard relies heavily on count on the initial browser load. I've never personally used ANALYZE on a large dataset before so I'm not sure of the performance.

For the dashboard, I think we can add some documentation to recommend users to enable `autovalcuum` and its threshold if they are using Postgres.